### ISSUE TITLE
Bambu Studio - Import STEP files

### ISSUE THREAD

**Issue by kvak1** [bambulab/BambuStudio#5240]
### Bambu Studio Version

1.10.0.89

### Where is the application from?

Bambu Lab Official website

### OS version

win 11

### Additional system information

Cpu i5, memory 32 Gb

### Printer

P1S

### How to reproduce

If I try import any STEP file, the studio opens form "Step file important  parameters".  After choosing any action from the choices of the form the application waits - calculates and in the end finisches with information "Unhandled unknown exception; terminating the application. If I refuse correcting the file file and just press "Cancel" no file is imported. Other types of files like stl and 3mf work normaly.

### Actual results

Application works normaly but I have to use stl or 3mf files.

### Expected results

I need to work with STEP files.

### Project file & Debug log uploads

I hope no file is not needed..

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by Trobinou** [comment]
I have the same bug on Windows 10, except that after clicking on the "OK" button in the error message, Bambu Studio closes automatically.

**Comment by kvak1** [comment]
Yes, I have the same too, I forgot mention it. I realized another thing - if you switch of the choice "Show the step mesh parameter setting dialog" in General setting (ctrl-p), the application shows the message about its terminating directly without offering correcting your step file. It's sad but I am glad that is not my mistake. I hope it will be repaired soon. Thanks..

**Comment by DoTsTeR77** [comment]
yes, major issues with me importing step files that previously imported fine.. the beta had the same problem.. I had to revert back to previous version..

**Comment by MartinChan3** [comment]
It seems that Studio has released a temporary package to provide an interim solution for those experiencing issues on Mac (Intel chip), STEP import crashes, and missing temperature displays. Hopefully, they will release the actual hotfix soon!
Windows：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_win_public-v01.10.00.94-20241114223322.exe
Mac：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_mac-v01.10.00.94-20241114223322.dmg

**Comment by kvak1** [comment]
> It seems that Studio has released a temporary package to provide an interim solution for those experiencing issues on Mac (Intel chip), STEP import crashes, and missing temperature displays. Hopefully, they will release the actual hotfix soon! Windows：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_win_public-v01.10.00.94-20241114223322.exe Mac：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_mac-v01.10.00.94-20241114223322.dmg

Thank you very much Martin, I have just installed version (94) from your link. Steps work and temperature is visible (I didnt noticed that before). Yes I hope too that new release will be published soon but for this moment I am happy that I can use my steps. Karel

**Comment by DoTsTeR77** [comment]
yup, just tried the new  package.. I still cant import some step files no matter how much I change those sliders.. I'd love to see it back the way it was.. or put an option to import them the way the previous version did which gave me no issues..

ohh.. Could I ask what those sliders values would have been by default if the previous version used them?

Also.. all work on this is appreciated.. I dont want to sound like it isnt..

**Comment by DoTsTeR77** [comment]
> It seems that Studio has released a temporary package to provide an interim solution for those experiencing issues on Mac (Intel chip), STEP import crashes, and missing temperature displays. Hopefully, they will release the actual hotfix soon! Windows：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_win_public-v01.10.00.94-20241114223322.exe Mac：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_mac-v01.10.00.94-20241114223322.dmg

yup, just tried the new package.. I still cant import some step files no matter how much I change those sliders.. I'd love to see it back the way it was.. or put an option to import them the way the previous version did which gave me no issues..

Unfortunately I cant provide the step files as it work under NDA..

ohh.. Could I ask what those sliders values would have been by default if the previous version used them?

Also.. all work on this is appreciated.. I dont want to sound like it isnt..

**Comment by kvak1** [comment]
> > It seems that Studio has released a temporary package to provide an interim solution for those experiencing issues on Mac (Intel chip), STEP import crashes, and missing temperature displays. Hopefully, they will release the actual hotfix soon! Windows：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_win_public-v01.10.00.94-20241114223322.exe Mac：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_mac-v01.10.00.94-20241114223322.dmg
> 
> yup, just tried the new package.. I still cant import some step files no matter how much I change those sliders.. I'd love to see it back the way it was.. or put an option to import them the way the previous version did which gave me no issues..
> 
> Unfortunately I cant provide the step files as it work under NDA..
> 
> ohh.. Could I ask what those sliders values would have been by default if the previous version used them?
> 
> Also.. all work on this is appreciated.. I dont want to sound like it isnt..

I've just open the Studio and new version 1.10.1 is offered. Try it. There are more changes in it...

**Comment by DoTsTeR77** [comment]
> > > It seems that Studio has released a temporary package to provide an interim solution for those experiencing issues on Mac (Intel chip), STEP import crashes, and missing temperature displays. Hopefully, they will release the actual hotfix soon! Windows：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_win_public-v01.10.00.94-20241114223322.exe Mac：https://github.com/bambulab/BambuStudio/releases/download/v01.10.00.89/Bambu_Studio_mac-v01.10.00.94-20241114223322.dmg
> > 
> > 
> > yup, just tried the new package.. I still cant import some step files no matter how much I change those sliders.. I'd love to see it back the way it was.. or put an option to import them the way the previous version did which gave me no issues..
> > Unfortunately I cant provide the step files as it work under NDA..
> > ohh.. Could I ask what those sliders values would have been by default if the previous version used them?
> > Also.. all work on this is appreciated.. I dont want to sound like it isnt..
> 
> I've just open the Studio and new version 1.10.1 is offered. Try it. There are more changes in it...

I actually did try 1.10.1 as well, same day I posted what I posted.. same behavior..
